pkg/kubelet/volumes.go

```diff
@@ -285,10 +285,6 @@ func (kl *Kubelet) newVolumeAttacherFromPlugins(spec *volume.Spec, pod *api.Pod,
 	if err != nil {
 		return nil, fmt.Errorf("can't use volume plugins for %s: %v", spec.Name(), err)
 	}
-	if plugin == nil {
```

Comment:
I think it is okay to remove this because FindPluginBySpec does not return nil,nil. If you did not get an error you are getting a plugin.